pkg/templates/templates_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package templates
```

Comment:
Why is this not a part of `pkg/templates/partitions_test.go`?

Reply:
> Why is this not a part of `pkg/templates/partitions_test.go`?

Just for future expansion. As we might add more test file in this package.
But we can extract it also later on if looks clearer.